spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/web/client/BasicAuthHttpHeaderProvider.java

```diff
@@ -25,6 +25,7 @@
 import org.springframework.util.Base64Utils;
 import org.springframework.util.StringUtils;
 
+import de.codecentric.boot.admin.server.config.AdminServerProperties.InstanceCredentials;
```

Comment:
I'd suggest to move the imported class to the `BasicAuthHttpHeaderProvider` class

Reply:
Moved class